mappings/net/minecraft/nbt/NbtInt.mapping

```diff
@@ -1,4 +1,7 @@
-CLASS net/minecraft/class_2497 net/minecraft/nbt/IntTag
+CLASS net/minecraft/class_2497 net/minecraft/nbt/NbtInt
+	COMMENT Represents a NBT 32-bits integer.
```

Comment:
```suggestion
	COMMENT Represents a NBT 32-bit integer.
```

Reply:
I always get it wrong, then learn about it and still manage to forget it. Sorry! Should be fixed now.